src/main/java/xyz/groundx/caver_ext_kas/wallet/KASWallet.java

```diff
@@ -55,7 +55,18 @@
      * @param walletAPI An WalletAPI instance to use KAS Wallet API.
      */
     public KASWallet(Wallet walletAPI) {
+        this(walletAPI, false);
+    }
+
+    /**
+     * Creates a wallet instance that uses the KAS Wallet API. <br>
+     * KASWallet used as a member of CaverExtKAS named wallet.
+     * @param walletAPI An WalletAPI instance to use KAS Wallet API.
+     * @param nodeApiInitialized A boolean to check whether the NodeApi is initialized or not.
+     */
+    public KASWallet(Wallet walletAPI, boolean nodeApiInitialized) {
```

Comment:
Why does need this constructor?

Reply:
In most case, I think it will be used for internal purposes only.
So I think it should be private.

Thanks for your feedback. I'll update it.